Authentication

akankshadhanewar opened the pull request.

Commit:
completed below tasks:
Task 1 - Authenticate user and show user home
Task 2 - Allow authenticated user to home and logout
Task 3 - Import Books

Commit:
Completed authentication module

rishikaharithaMSIT commented on application.py:
```diff
@@ -1,26 +1,36 @@
+app.secret_key="email"
+
+db.init_app(app)
+def main():
+    db.create_all()
```
create db should be in separate file

akankshadhanewar replied:
changes have been modified. Please check.

Commit:
Changes to be committed:
    new file:   __pycache__/models.cpython-35.pyc
    modified:   application.py
    new file:   main.py
